LICENSE.txt

```diff
@@ -74,6 +74,9 @@ File: skimage/feature/fisher_vector.py
 Copyright: 2014 2014 Dan Oneata
 License: MIT
 
+File: skimage/transform/_thin_plate_splines.py
+Copyright: 2007 Zachary Pincus
+License: GNU General Public License v2.0
```

Comment:
```suggestion
Copyright: 2007 Zachary Pincus
           2023 the scikit-image team
License: BSD-3-Clause
```
The author agreed to relicense with the one used by scikit-image itself. I don't think you can transfer copyright, and as a courtesy we should keep the copyright of the original author. But I think we can indicate this way that we modified the algorithm. 

cc @stefanv, you are usually knowledgeable in this stuff.